vignettes/xcms.Rmd

```diff
@@ -207,7 +207,7 @@ binned base peak chromatograms.
 bpis_bin <- bin(bpis, binSize = 2)
 
 ## Calculate correlation on the log2 transformed base peak intensities
-cormat <- cor(log2(do.call(cbind, lapply(bpis_bin, intensity))))
+cormat <- cor(dplyr::bind_cols(lapply(bpis_bin, intensity)))
```

Comment:
Hi, saw the PR only now. So you intend to replace `do.call(cbind)`
with `bind_cols()`, but your patch is dropping the log2. Yours, Steffen

Reply:
Also, I would prefer a patch that does not add new dependencies - and some information or example data why/where the original code failed